.github/workflows/ci.yml

```diff
@@ -12,7 +12,17 @@ jobs:
         operating-system: ["ubuntu-latest"]
         php-versions: ["7.4", "8"]
 
+    env:
+      DB_DATABASE: nova_pages_tool_testing
+      DB_USER: root
+      DB_PASSWORD: root
+
     steps:
+      - name: Setup MySQL
+        run: |
+          sudo /etc/init.d/mysql start
+          mysql -e 'CREATE DATABASE ${{ env.DB_DATABASE }};' -u${{ env.DB_USER }} -p${{ env.DB_PASSWORD }}
```

Comment:
Ik wist niet dat MySQL standaard in de Ubuntu installatie zit. Je kan ook `services` gebruiken dan wordt een Docker container opgestart. Effect is hetzelfde, maar je kan dan wel kiezen welke MySQL versie je gebruikt. Nu moet je het met de versie doen die Ubuntu levert.